src/css/profile/mobile/common/checkbox.less

```diff
@@ -11,92 +11,40 @@ input[type="checkbox"].ui-checkbox:not(.ui-toggle-switch) {
 
 input[type="checkbox"].ui-checkbox:not(.favorite) {
 
-	&::before,
-	&::after {
+	&::before {
 		content: "";
 		position: absolute;
-		width: 100%;
-	}
-
-	&::before {
-		height: 100%;
-		opacity: 0.8;
-		background-color: var(--control-inactive-color);
-		mask-image: url("images/3_Controllers/Checkbox/sem_checkedtextview_check_to_on_mtrl_000.svg");
-		mask-size: 100%, 0;
-		mask-position: center;
-		mask-repeat: no-repeat;
-	}
-
-	&:checked::before {
-		mask-size: 100%, 100%;
+		left: 0;
+		top: 0;
+		width: 32 * @px_base;
+		height: 32 * @px_base;
+		background-color: var(--ripple-color);
+		border-radius: 100%;
+		opacity: 0;
 	}
 
 	&::after {
-		height: 0;
+		content: "";
+		position: absolute;
+		width: 100%;		
```

Comment:
width: 100% is duplicated, see 3 lines below

Reply:
Removed.